### DISCUSSION TITLE
Alt key equivalent on macOS

### DISCUSSION THREAD

**Discussion by OrkunTokdemir** [PatrickF1/fzf.fish#233] [Q&A] [answered]
Hey, 

I would like to use shortcuts including the alt key on macOS but I couldn't find the equivalent of it. How can I use it or add the command key to the shortcuts on macOS?  Could you please help me? 
```
        FEATURE            |  MNEMONIC KEY SEQUENCE        |  CORRESPONDING OPTION
        Search directory   |  Ctrl+Alt+F (F for file)      |  --directory
        Search git log     |  Ctrl+Alt+L (L for log)       |  --git_log
        Search git status  |  Ctrl+Alt+S (S for status)    |  --git_status
        Search history     |  Ctrl+R     (R for reverse)   |  --history
        Search variables   |  Ctrl+V     (V for variable)  |  --variables
        Search processes   |  Ctrl+Alt+P (P for process)   |  --processes
```

**Comment by PatrickF1** [comment, marked as answer]
Alt is option. I don't think the Command key can be used as a shortcut for fish.

**Reply by OrkunTokdemir** [reply]
Thank you @PatrickF1  for the quick reply. 

The option key does not send the same signal as Alt. At least, it doesn't work on my end. How can I add the shift key to shortcuts? 

```
fzf_configure_bindings --git_status=\c<shit key symbol>s
```
What should I put into  `<shit key symbol>`?

**Reply by PatrickF1** [reply]
You can't use shift for fish key bindings. I forgot how I know that but it's somewhere in the fish docs if you look for it. Btw this question is not particular to fzf.fish but to fish shell so I'd ask in that space.

**Reply by PatrickF1** [reply]
Found out why shift doesn't work here https://fishshell.com/docs/current/cmds/bind.html#terminal-limitations.

**Reply by OrkunTokdemir** [reply]
@PatrickF1  Thank you for your help 🙂

**Reply by kidonng** [reply]
> I don't think the Command key can be used as a shortcut for fish.

Fish does support it, but the actual availability depends on your terminal. Quoting from the manual page you posted:

> Other keys don’t have a direct encoding, and are sent as escape sequences. For example <kbd>→</kbd> (Right) often sends `\e\[C`. These can differ from terminal to terminal

For example on kitty, `fish_key_reader` reports <kbd>Command</kbd> + <kbd>a</kbd> as `\e\[97\;9u`.

**Comment by judahrand** [comment]
I'm having a similar issue. `fish_key_read` does not detect `Control + Option + L` as anything.

**Reply by PatrickF1** [reply]
This might be helpful https://github.com/PatrickF1/fzf.fish/discussions/97.
